railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults.rb.tt

```diff
@@ -7,6 +7,8 @@
 #
 <%- end -%>
 # Read the Guide for Upgrading Ruby on Rails for more info on each option.
+
+Rails.application.config.raise_on_unpermitted_parameters = <%= options[:update] ? false : true %>
```

Comment:
I think this should be `true` for update too. This template is for upgrading 4.2 applications: we're only worried about keeping the existing behaviour for applications already on 5.0.

Reply:
Could you explain again why this should be `true` when running `rails app:update` right after upgrading from Rails 4.2?
From the comment above, that means I am supposed to flip it to `false` after I have made sure it works with `false`. But that doesn't make sense.